sample/Saturn.Sample/Saturn.Sample.fs

```diff
@@ -106,12 +107,21 @@ let userController = controller {
     edit (fun (ctx, id) -> (sprintf "Edit handler - %s" id) |> Controller.text ctx)
 }
 
+let dumpSession (session: ISession) =
+  session.Keys
+  |> Seq.map (fun k ->
+    sprintf "%s:%A" k (session.Get(k))
+  )
+  |> String.concat "\n"
+
 //Since all computation expressions produces `HttpHandler` everything can be easily composed together in nice declarative way.
 //I belive that aim of the Saturn should be providing a more streamlined, higher level developer experiance on top of the great
 //Giraffe's model. It's bit like Phoenix using Plug model under the hood.
 
 let topRouter = scope {
     pipe_through headerPipe
+    pipe_through fetchSession
```

Comment:
I guess we can keep that, since in most cases I'd call it good practice to have fetchSession on top router level.

Reply:
eh, I think I'll remove it, just because there's nothing else in this sample that's using session. In the previous version of the csrf work, `fetchSession` was a way to make sure that session was available when we used it to validate the CSRF tokens, but that's now handled by the Antiforgery feature that we're delegating to.